.github/workflows/php.yml

```diff
@@ -52,6 +50,11 @@ jobs:
         if: ${{ steps.paths.outputs.phpcs == 'true' }}
         run: parallel-lint --exclude .git --exclude .github --exclude vendor .
 
+      - name: Fetch base and head branches
+        run: |
+          git fetch origin ${{ github.base_ref }}:${{ github.base_ref }}
+          git fetch origin ${{ github.head_ref }}:${{ github.head_ref }}
```

Comment:
:thinking: Does this speed up the CI pipeline by only checking out the branches needed?

Reply:
I believe that fetching only necessary branches can make CI pipeline to run faster, specially on larger repositories. But it's not just about speed, I think it's more semantic tough.